### DISCUSSION TITLE
leaning pooltable

### DISCUSSION THREAD

**Discussion by mcblockserrilla** [FairlySadPanda/vrcbce#178] [Q&A] [answered]
making a $h!t bar world for vrc, and want to make at least the Q ball curve a little after its hit.
like the table is leaning to one side. 
any idea how i could do this?

**Comment by Metamaniac** [comment, marked as answer]
The physics currently doesn't allow for there to be any "forces" externally applied to the simulation, this would probably be doable with the planned re-write... though I honestly don't know ***why*** you'd want this.
